File: frontend/models/MessagingLog.ts
class MessagingLog
{
    member_uuid!: string;
    room_uuid!: string;
    timestamp!: string;
    _id?: string;

    constructor({ _id, member_uuid, room_uuid, timestamp}: MessagingLog){
        this._id = _id;
        this.member_uuid = member_uuid;
        this.room_uuid = room_uuid;
        this.timestamp = timestamp;
    }
}

export default MessagingLog;

File: log-service/Models/MessagingLog.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace log_service.Models
{

    public class MessagingLog
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string? Id { get; set; }
        public string room_uuid { get; set; } = null!;
        public string member_uuid { get; set; } = null!;
        public string timestamp { get; set; } = null!;
    }
}